lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.CoilCoolingDXSingleSpeed.rb

```diff
@@ -0,0 +1,97 @@
+class ASHRAE901PRM < Standard
+  # @!group CoilCoolingDXSingleSpeed
+
+  include ASHRAEPRMCoilDX
+
+  # Finds capacity in W
+  #
+  # @return [Double] capacity in W to be used for find object
+  def coil_cooling_dx_single_speed_find_capacity(coil_cooling_dx_single_speed, sys_type)
```

Comment:
The methods in this file are missing YARD comments for the parameters.

Reply:
Added